src/Components/Pages/RentStabilization/RentStabilization.tsx

```diff
@@ -0,0 +1,123 @@
+                Good Cause Eviction law because you already have stronger
+                existing protections through rent stabilization law.
+              </p>
+            </div>
+          </div>
+        </ContentBox>
+
+        <ContentBox
+          headerTitle="WHAT YOU CAN DO"
+          headerSubtitle="How to find out if your apartment is rent stabilized"
+        >
+          <div className="content-box__section">
+            <div className="content-box__section__content">
+              <div className="content-box__section__header">
+                Check your lease
+              </div>
+              <p>
+                You can find out if your apartment is rent stabilized by
+                checking your lease. If your lease says you're apartment is rent
+                stabilized, then it's safe to say it is! We still recommend
+                requesting your rent history to be extra certain (see below),
+                especially if your lease does not mention anything about rent
+                stabilization.
+              </p>
+            </div>
+          </div>
+
+          <div className="content-box__section">
+            <div className="content-box__section__content">
+              <div className="content-box__section__header">
+                Inquire through Homes and Community Renewal
+              </div>
+              <p>
+                After submitting your request, if you get an email the next day
+                saying your apt doesn't have a registration on file, that means
+                you're apartment has never been rent stabilized, and therefore
+                you meet the rent regulation criteria for Good Cause Eviction
+              </p>
+              <JFCLLinkExternal href="https://portal.hcr.ny.gov/app/ask">
+                Fill out the form
+              </JFCLLinkExternal>
+            </div>
+          </div>
+
+          <div className="content-box__section">
+            <div className="content-box__section__content">
+              <div className="content-box__section__header">
+                Request your apartment’s rent history
+              </div>
+              <p>
+                Use JustFix’s rent history request tool to help find out if your
+                apartment is rent stabilized and if you're being overcharged. If
+                your apartment has ever been rent stabilized, you’ll receive a
+                document showing the rents that your landlord has registered
+                since 1984.
+              </p>
+              <JFCLLinkExternal href="https://app.justfix.org/rh">
+                Start my request
+              </JFCLLinkExternal>
+            </div>
+          </div>
+
+          <div className="content-box__footer">
+            <div className="content-box__section__content">
+              <div className="content-box__section__header">
+                Learn more about your rights
```

Comment:
should this content be "Have you learned new information about your apartment?" like it is on the results page?